composer.json

```diff
@@ -47,7 +47,12 @@
     },
     "config": {
         "optimize-autoloader": true,
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "composer/package-versions-deprecated": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "infection/extension-installer": true
```

Comment:
Also not needed: we should run with `--no-interaction` instead

Reply:
If I run `composer install` on this project (for development) I get this:
```
❯ composer i
infection/extension-installer contains a Composer plugin which is currently not in your allow-plugins config. See https://getcomposer.org/allow-plugins
Do you trust "infection/extension-installer" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?]
```

That's what the PR is about. Or should it just say `false`?

Reply:
You can sat "no" and let it continue, or you can run with `--no-interaction` yourself too.

Reply:
If I say no, it updates the composer.json file with `"infection/extension-installer": false`. 
I can say discard, but then it will ask me next time.

Reply:
Let it ask - it's no big deal, but it really shouldn't be necessary in here: it's just yet another transitive dependency declared explicitly (where it shouldn't be)